jackal_control/launch/control.launch

```diff
@@ -1,10 +1,10 @@
 <launch>
-  <rosparam command="load" file="$(find jackal_control)/config/control.yaml" />
+  <rosparam file="$(find jackal_control)/config/control.yaml" command="load"/>
 
   <node name="controller_spawner" pkg="controller_manager" type="spawner"
         args="jackal_joint_publisher jackal_velocity_controller --shutdown-timeout 1" />
-  
-  <node pkg="robot_localization" type="ekf_localization_node" name="ekf_localization" /> 
+
+  <include file="$(find jackal_control)/launch/robot_localization.launch"/>
```

Comment:
Is it necessary for this to be included in `control.launch`?

If it's going to be separate anyway, perhaps it should just be its own top-level launch file on the robot? Necessary changes would be:

https://github.com/jackal/jackal_simulator/blob/indigo-devel/jackal_gazebo/launch/jackal_world.launch#L31
https://github.com/jackal/jackal_robot/blob/indigo-devel/jackal_base/launch/base.launch#L38

A lot of nested includes are harder to reason about; would prefer avoiding that. Thoughts?